js/localization/globalize/date.js

```diff
@@ -709,11 +714,11 @@ if(Globalize && Globalize.formatDate) {
             let formatCacheKey;
 
             if(typeof (format) === 'function') {
-                return format(date);
+                return this.removeRtlMarks(format(date));
```

Comment:
In this case format is a custom function. I don't think we should modify it's result.

Reply:
got it, thanks